### PR TITLE
add prettier

### PR DESCRIPTION
**Pull request by ktalebian**
* Adds prettier support

### CODE REVIEW

File: packages/eslint-config-twilio/.eslintrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "./index.js"
+}
```
Comment (ahcai) [inline packages/eslint-config-twilio/.eslintrc]: missing line at EOF

File: packages/eslint-config-twilio/package.json [hunk 2]
```diff
@@ -26,10 +26,19 @@
   ],
   "main": "index.js",
   "scripts": {
+    "check-duplicate": "eslint --print-config index.js | eslint-config-prettier-check",
     "test": "jest --coverage --color"
   },
+  "dependencies": {
+    "eslint-config-twilio-base": "file:///Users/ktalebian/Projects/github/twilio-labs/twilio-style/packages/eslint-config-twilio-base"
```
Comment (ahcai) [inline packages/eslint-config-twilio/package.json]: 😂

File: packages/eslint-config-twilio/rules/base.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  'template-tag-spacing': 'off',
+  'unicode-bom': 'off',
+  'wrap-iife': 'off',
+  'yield-star-spacing': 'off',
+  'arrow-body-style': 'off',
+  'max-len': 'off',
+  'prefer-arrow-callback': 'off',
+  'no-mixed-operators': 'off',
+  'quotes': 'off',
+};
```
Comment (ahcai) [inline packages/eslint-config-twilio/rules/base.js]: missing line at EOF

File: packages/eslint-config-twilio/rules/index.js [hunk 4]
```diff
@@ -1,9 +1,5 @@
...
-    require('./es6'),
-    require('./node'),
-    require('./possible-errors'),
-    require('./stylistic-issues'),
-    require('./variables'),
-);
+  {},
+  require('./base'),
+  require('./prettier'),
+);
```
Comment (ahcai) [inline packages/eslint-config-twilio/rules/index.js]: same

File: packages/eslint-config-twilio/rules/prettier.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  'prettier/prettier': [
+    'error',
+    {
+      'trailingComma': 'all',
+      'singleQuote': true,
+    },
+  ]
+}
```
Comment (ahcai) [inline packages/eslint-config-twilio/rules/prettier.js]: here too

File: packages/eslint-config-twilio-base/rules/stylistic-issues.js [hunk 6]
```diff
@@ -105,7 +105,16 @@ module.exports = {
...
+      beforeBlockComment: true,
+      afterBlockComment: true,
+      beforeLineComment: true,
+      afterLineComment: true,
+      allowBlockStart: true,
+      allowBlockEnd: true,
+      allowObjectStart: true,
+      allowObjectEnd: true,
+      allowArrayStart: true,
+      allowArrayEnd: true
```
Comment (ahcai) [inline packages/eslint-config-twilio-base/rules/stylistic-issues.js]: missing trailing comma

File: packages/eslint-config-twilio/rules/base.js [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+  'space-infix-ops': 'off',
+  'space-unary-ops': 'off',
+  'switch-colon-spacing': 'off',
+  'template-curly-spacing': 'off',
+  'template-tag-spacing': 'off',
+  'unicode-bom': 'off',
+  'wrap-iife': 'off',
+  'yield-star-spacing': 'off',
+  'arrow-body-style': 'off',
+  'max-len': 'off',
```
Comment (ahcai) [inline packages/eslint-config-twilio/rules/base.js]: hm, what will this mean for our max-len override?

Comment (ktalebian) [reply packages/eslint-config-twilio/rules/base.js]: All add it to prettier

**Comment by vnguyen94** [comment]
could be meta and lint the repo by using the rules from the subpackage?

**Pull request merged**